kubemarine/resources/configurations/defaults.yaml

```diff
@@ -42,7 +42,7 @@ services:
     # for patches functionality we need v1beta3 which is available since k8s v1.22 only
     apiVersion: '{% if "v1.21" in services["kubeadm"]["kubernetesVersion"] %}kubeadm.k8s.io/v1beta2{% else %}kubeadm.k8s.io/v1beta3{% endif %}'
     kind: ClusterConfiguration
-    kubernetesVersion: v1.24.11
+    kubernetesVersion: v1.26.3
```

Comment:
Please also update schema/services/kubeadm.json. Also, I would like to remind that every update of defaults changes Kubemarine behaviour which can (highly unlikely) break some usages. https://github.com/Netcracker/KubeMarine/pull/345#discussion_r1100194109

Reply:
update json schema